boxfile.yml

```diff
@@ -1,59 +1,76 @@
+# This boxfile.yml is designed to generate a Laravel project when first run.
+# If you have an existing Laravel project and are using this as a template,
+# Remove lines 38 and 39 and uncomment lines 36 and 37 (the npm commands)
+
 run.config:
-  # install php and associated runtimes
   engine: php
-
-  # php engine configuration (php version, extensions, etc)
   engine.config:
-
-    # sets the php version to 7.0
-    runtime: php-7.0
-
-    # use nginx
+    runtime: php-7.1
     webserver: nginx
-
-    # public directory
     document_root: public
-
-    # enables php extensions
     extensions:
       - pdo
-      - pdo_mysql
       - mbstring
       - tokenizer
       - session
       - zip
       - dom
       - xml
       - ctype
+      - xmlwriter
+      - pdo_mysql
 
-#
-deploy.config:
-  before_live:
-    web.main:
-      - php artisan migrate --force
-      - mkdir -p storage/framework/{sessions,cache,views}
+  # Use node/npm locally for mix
```

Comment:
Remove reference to Elixir?

Reply:
Mix is actually a Laravel asset compiler packaged with 5.5: https://laravel.com/docs/5.5/mix